adapters/adyoulike/adyoulike_test.go

```diff
@@ -0,0 +1,246 @@
+package adyoulike
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/stretchr/testify/assert"
+)
+
+const testsBidderEndpoint = "https://localhost/bid/4"
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAdyoulike, config.Adapter{
+		Endpoint: testsBidderEndpoint})
+
+	if buildErr != nil {
+		t.Fatalf("Builder returned unexpected error %v", buildErr)
+	}
+
+	adapterstest.RunJSONBidderTest(t, "adyouliketest", bidder)
+}
+
+func TestMakeRequestNoImp(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderAdyoulike, config.Adapter{
+		Endpoint: testsBidderEndpoint})
+
+	assert.Nil(t, buildErr, "buildErr must be nil")
+
+	var reqInfo adapters.ExtraRequestInfo
+	var req openrtb.BidRequest
+	req.ID = "test_id"
+
+	bids, errs := bidder.MakeRequests(&req, &reqInfo)
+
+	assert.EqualError(t, errs[0], "No impression in the bid request")
+	assert.Len(t, bids, 0)
+}
+
+func TestMakeRequestInvalidParams(t *testing.T) {
```

Comment:
Other adapters test param validation in a separate `adapters/fooadapter/params_test.go`. Consider including one for your adapter